src/concurrent_interner.rs

```diff
@@ -413,7 +413,7 @@ impl ConcurrentInternerMemberStorage {
 
     // Might as well save 24 bytes elsewhere as well. 🤷🏽
     #[cfg(not(all(target_arch = "aarch64", any(target_os = "macos", target_os = "ios"))))]
-    const STORAGE_CHUNK_SIZE: usize = 4192 - 24;
+    const STORAGE_CHUNK_SIZE: usize = 4096 - 24;
```

Comment:
YMMV: you could also write these in non-decimal to help avoid these typos.

For example, it could be 
```suggestion
    const STORAGE_CHUNK_SIZE: usize = (1 << 12) - 24;
```
or
```suggestion
    const STORAGE_CHUNK_SIZE: usize = 0x1000 - 24;
```

Reply:
I find the decimal representation more understandable at a quick glance. It'd be a different story if we had lots of these constants, but there's only a couple, so I think it's fine.